config/webpack/tests.js

```diff
@@ -5,5 +5,5 @@ const { merge } = require('webpack-merge');
 const sharedConfig = require('./shared');
 
 module.exports = merge(sharedConfig, {
-  mode: 'development',
+  mode: 'production',
```

Comment:
What does this change exactly?

Reply:
In `development`, Webpack uses `eval` to handle dynamic imports and other features, which requires a CSP change

Reply:
Oh, right, that makes sense!